frontend/src/pages/ActivityReport/Pages/components/GoalForm.js

```diff
@@ -164,6 +164,7 @@ export default function GoalForm({
 
   const prompts = combinePrompts(templatePrompts, goal.prompts);
   const isCurated = goal.isCurated || false;
+  const canEditFei = status !== 'Closed';
```

Comment:
Not sure if its possible to have a closed goal on a open AR.

Reply:
right now "goalsForGrants" excludes closed and suspended goals

Reply:
Ok so from this screen we can hard code true, correct?

Reply:
think so